meson.build

```diff
@@ -81,8 +81,12 @@ endif
 
 if cc.has_argument('--std=gnu99')
   add_project_arguments('--std=gnu99', language: ['c', 'cpp'])
+  lang_opt = ['c_std=gnu99']
 elif cc.has_argument('--std=c99')
   add_project_arguments('--std=c99', language: ['c', 'cpp'])
+  lang_opt = ['c_std=c99']
+else
+  lang_opt = []
```

Comment:
better to not do this. i prefer the c_std flags to be directly defined in `subproject` also create a PR with a branch name containing the word `dist` (required to run all the CIs) and close this and the other rebase pr. you can rebase in one PR directly.

Reply:
PR with new branch name has been created: https://github.com/rizinorg/rizin/pull/4684